tests/modules/appointments/services/service.spec.ts

```diff
@@ -0,0 +1,459 @@
+// createAppointment.test.js
```

Comment:
Remove unnecessary comments, also update the filename for this. `appointment.service.spec.ts